cypress/support/e2e.ts

```diff
@@ -17,6 +19,7 @@ declare global {
     }
     interface VisitOptions {
       serviceWorker?: true
+      featureFlag?: FeatureFlag
```

Comment:
Amend to `featureFlags?: FeatureFlag[]`, so that it is possible to test with multiple flags at once.
You'll also need to update the logic in L44:

    const featureFlags = options.featureFlags.reduce((flags, flag) => {
      ...flags,
      [flag]: 'enabled',
    }, {})
    win.localStorage.setItem('featureFlags', featureFlags)